pkgs/development/python-modules/theano-pymc/default.nix

```
@@ -0,0 +1,47 @@
+{ lib
+, fetchPypi
+, buildPythonPackage
+, pythonOlder
+, pandas
+, numpy
+, scipy
+, filelock
+, pytest
+, nose
+, parameterized
+}:
+
+buildPythonPackage rec {
+  pname = "theano-PyMC";
+  version = "1.1.2";
+  disabled = pythonOlder "3.6";
+
+  src = fetchPypi {
+    inherit pname version;
```

Looks like this is case sensitive for leading T
```suggestion
    pname = "Theano-PyMC";
    inherit version;
```

```
       > trying https://pypi.io/packages/source/t/theano-PyMC/theano-PyMC-1.1.2.tar.gz
       >   % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
       >                                  Dload  Upload   Total   Spent    Left  Speed
       > 100   122  100   122    0     0   1881      0 --:--:-- --:--:-- --:--:--  1906
       > 100   282  100   282    0     0   2208      0 --:--:-- --:--:-- --:--:--  2208
       >   0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
       > curl: (22) The requested URL returned error: 404
       > error: cannot download theano-PyMC-1.1.2.tar.gz from any mirror
       For full logs, run 'nix log /nix/store/khxqpfpnqwflfkyw3fzkkyhrz8nv1a9d-theano-PyMC-1.1.2.tar.gz.drv'.